### PR TITLE
Updates for Musicoin EIP-150 and EIP-155 enable configuations.

### PR DESCRIPTION
**Pull request by trustfarm-dev**
1. EIP-150 : gas ddos preventing configuration
  activated block number : 1900001  [seamless transition is possible]
  compatible with non-eip 150 nodes.
2. EIP-155 : replay attack prevention.
  activated block number : 2000001  [block will regenerated from 2000001]
  after that, sendTransaction only works with EIP155 enabled node.

3. Musicoin Test Node setup configuration.
  We need TestNet. it is started from 0 block.

### CODE REVIEW

**Comment by 5chdn** [comment]
Can we work on top of #62 ?

File: params/config.go [hunk 1]
```diff
@@ -59,7 +59,7 @@ var (
 		EIP150Hash:     common.HexToHash("0x41941023680923e0fe4d74a34bdac8141f2540e3ae90623718e47d66d1ca4a2d"),
```
Comment (5chdn) [inline params/config.go]: This will probably break syncing

File: params/config.go [hunk 2]
```diff
@@ -59,7 +59,7 @@ var (
...
 		EIP155Block:    big.NewInt(10),
 		EIP158Block:    big.NewInt(10),
-		ByzantiumBlock: big.NewInt(1700000),
+		ByzantiumBlock: big.NewInt(36028797018963967),
```
Comment (5chdn) [inline params/config.go]: Why are we disabling this?

**Comment by trustfarm-dev** [comment]
I think, bzt fork will applied on this fork?
also, top of #62  and changed block height is also good choice.

**Comment by 5chdn** [comment]
Merged into #62

**Pull request closed without merging**